src/chttpd/src/chttpd.erl

```diff
@@ -54,74 +90,91 @@
     code,
     headers,
     chunks,
-    resp=nil,
-    buffer_response=false
+    resp = nil,
+    buffer_response = false
 }).
 
 start_link() ->
     start_link(http).
 start_link(http) ->
     Port = config:get("chttpd", "port", "5984"),
     start_link(?MODULE, [{port, Port}]);
-
 start_link(https) ->
     Port = config:get("ssl", "port", "6984"),
     {ok, Ciphers} = couch_util:parse_term(config:get("ssl", "ciphers", "undefined")),
     {ok, Versions} = couch_util:parse_term(config:get("ssl", "tls_versions", "undefined")),
-    {ok, SecureRenegotiate} = couch_util:parse_term(config:get("ssl", "secure_renegotiate", "undefined")),
+    {ok, SecureRenegotiate} = couch_util:parse_term(
+        config:get("ssl", "secure_renegotiate", "undefined")
+    ),
     ServerOpts0 =
-        [{cacertfile, config:get("ssl", "cacert_file", undefined)},
-         {keyfile, config:get("ssl", "key_file", undefined)},
-         {certfile, config:get("ssl", "cert_file", undefined)},
-         {password, config:get("ssl", "password", undefined)},
-         {secure_renegotiate, SecureRenegotiate},
-         {versions, Versions},
-         {ciphers, Ciphers}],
-
-    case (couch_util:get_value(keyfile, ServerOpts0) == undefined orelse
-        couch_util:get_value(certfile, ServerOpts0) == undefined) of
+        [
+            {cacertfile, config:get("ssl", "cacert_file", undefined)},
+            {keyfile, config:get("ssl", "key_file", undefined)},
+            {certfile, config:get("ssl", "cert_file", undefined)},
+            {password, config:get("ssl", "password", undefined)},
+            {secure_renegotiate, SecureRenegotiate},
+            {versions, Versions},
+            {ciphers, Ciphers}
+        ],
+
+    case
+        (couch_util:get_value(keyfile, ServerOpts0) == undefined orelse
```

Comment:
I would prefer the following here
```
(couch_util:get_value(keyfile, ServerOpts0) == undefined 
    orelse couch_util:get_value(certfile, ServerOpts0) == undefined)
```

OR
```
(
    couch_util:get_value(keyfile, ServerOpts0) == undefined 
    orelse 
    couch_util:get_value(certfile, ServerOpts0) == undefined
)
```

-1

Reply:
+2